guides/pull-requests.md

```diff
@@ -104,5 +105,13 @@ documentation.
 
 Any affected examples in your PR should have an appropriate label - `New`,
 `Updated`, or `Deprecated` - these are pulled from the `status` key within the
-Releases YAML file mentioned above. The appropriate indicator will then
-automatically be added to each page/UI element.
+[Releases YAML file](/releases.yml) mentioned above. The appropriate indicator
+will then automatically be added to the sidebar navigation.
+
+Additionally, you should manually add a status label to the appropriate
+heading on the docs page of each component you make changes to - this is done by
+using the [status label component](/scss/_patterns_status-label.scss).
```

Comment:
It may be nice to provide example labels for "New" "Updated" and "Deprecated" here